styles/styles.css

```diff
@@ -0,0 +1,131 @@
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  padding: 0 2rem;
+  font-family: "Work Sans", serif;
+}
+
+header {
+  background-image: url("../images/header-image.jpg");
+  background-size: cover;
+  color: #ffffff;
```

Comment:
It's a minor thing, but when using colors keep the same case across the files. My preference is all uppercase, but either is fine, but you use both all uppercase and all lower case in this file

Reply:
ooo, that's the copy and paste getting me. thanks!